packages/beacon-node/src/chain/historicalState/index.ts

```diff
@@ -8,6 +8,7 @@ import {
   HistoricalStateWorkerApi,
   HistoricalStateWorkerData,
 } from "./types.js";
+import {DiffLayers} from "./diffLayers.js";
```

Comment:
with new approach this file seems to be for a different purpose, need to update the class comment